go.mod

```diff
@@ -1,30 +1,58 @@
 module github.com/enfabrica/enkit
 
-go 1.14
+go 1.17
```

Comment:
Do we need to hang back at go 1.16 for appengine support still?

Reply:
Ups, changed to 1.16. Re ran gazelle and update-repos, no change.